### PR TITLE
Apply miscellaneous coding style fixes.

### PR DESCRIPTION
**Pull request by jonathanknowles**
Small adjustments to bring existing code into line with:
https://github.com/input-output-hk/cardano-wallet/wiki/Coding-Standards#limit-line-length-to-80-characters

### CODE REVIEW

File: src/Cardano/Wallet/BlockSyncer.hs [hunk 1]
```diff
@@ -54,7 +54,8 @@ tickingFunction getNextBlocks action tickTime = go
       go (BlockHeadersConsumed headersConsumed) = do
           blocksDownloaded <- getNextBlocks
           let blocksToProcess =
-                  filter (checkIfAlreadyConsumed headersConsumed) (L.nub blocksDownloaded)
+                  filter (checkIfAlreadyConsumed headersConsumed)
+                      (L.nub blocksDownloaded)
```
Comment (KtorZ) [inline src/Cardano/Wallet/BlockSyncer.hs]: Tiny note: when going for "multi-lining" a function call, I find it more readable to put a single argument per line, and a line break after the function name as well. e.g.:

```hs
let blocksToProcess = filter
        (checkIfAlreadyConsumed headersConsumed)
        (L.nub blocksDownloaded)
```

File: src/Cardano/Wallet/Primitive.hs [hunk 2]
```diff
@@ -241,7 +242,8 @@ instance Dom UTxO where
 
 balance :: UTxO -> Integer
 balance =
-    Map.foldl' (\total out -> total + fromIntegral (getCoin (coin out))) 0 . getUTxO
+    Map.foldl' (\total out -> total + fromIntegral (getCoin (coin out))) 0
+        . getUTxO
```
Comment (KtorZ) [inline src/Cardano/Wallet/Primitive.hs]: In such case, using an intermediate variable makes more sense to me.

```hs
balance =
    Map.foldl' fn 0 . getUTxO
  where
    fn :: Integer -> TxOut -> Integer
    fn total out = total + fromIntegral (getCoin (coin out))
```

**Review by KtorZ — APPROVED** [review]
A "neat-picky" PR but, okay.

File: src/Cardano/Wallet/Binary.hs [hunk 3]
```diff
@@ -79,9 +79,12 @@ import qualified Data.Set as Set
 
 decodeAddress :: CBOR.Decoder s Address
 decodeAddress = do
-    _ <- CBOR.decodeListLenCanonicalOf 2 -- CRC Protection Wrapper
-    tag <- CBOR.decodeTag -- Mysterious hard-coded tag cardano-sl seems to so much like
-    bytes <- CBOR.decodeBytes -- Addr Root + Attributes + Type
+    _ <- CBOR.decodeListLenCanonicalOf 2
+        -- ^ CRC Protection Wrapper
```
Comment (KtorZ) [inline src/Cardano/Wallet/Binary.hs]: The `^` is invalid here, this isn't a valid Haddock syntax :/

File: src/Cardano/Wallet/BlockSyncer.hs (hunk 1 above)
Comment (jonathanknowles) [reply src/Cardano/Wallet/BlockSyncer.hs]: I agree this is a good suggestion. I've updated my branch to match your suggestion.

File: src/Cardano/Wallet/Primitive.hs (hunk 2 above)
Comment (jonathanknowles) [reply src/Cardano/Wallet/Primitive.hs]: Thanks. Updated to match your suggestion.

File: src/Cardano/Wallet/Binary.hs (hunk 3 above)
Comment (jonathanknowles) [reply src/Cardano/Wallet/Binary.hs]: You're right! I've removed the `^`s.

**Pull request merged**